### PR TITLE
Updated servlet-5 and monitor-1 auto features

### PR DESCRIPTION
**Pull request by isaacrivriv**
- Enabled session monitor with servlet 5
- Transformed webcontainer monitor project for jakarta auto feature
- Added webcontainer monitor with servlet 5 auto feature

partial fix for #12259

### CODE REVIEW

**Comment by isaacrivriv** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.servlet5.0-monitor1.0.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.servlet5.0-monitor1.0
+Manifest-Version: 1.0
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.monitor-1.0))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.servlet-5.0))"
+IBM-Install-Policy: when-satisfied
+-bundles=com.ibm.ws.webcontainer.monitor.jakarta
+kind=ga
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.servlet5.0-monitor1.0.feature:9]: you can't go straight to GA ... start with noship and then move to beta

**Comment by isaacrivriv** [comment]
#build

**Comment by isaacrivriv** [comment]
#build

**Comment by isaacrivriv** [comment]
#build

**Comment by isaacrivriv** [comment]
#build

**Comment by isaacrivriv** [comment]
#run-libby-bot

**Review by cbridgha — APPROVED** [review]
LGTM

**Review by tevans78 — APPROVED** [review]

**Review by pnicolucci — APPROVED** [review]
LGTM

**Pull request merged**